packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -162,21 +162,21 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
             name: attendee.name,
             email: attendee.email,
             timeZone: attendee.timeZone,
-            language: { locale: attendee.locale || "" },
+            language: { locale: attendee.locale || "en" },
           };
         }),
         organizer: booking.user
           ? {
               name: booking.user.name || "",
               email: booking.user.email,
               timeZone: booking.user.timeZone,
-              language: { locale: booking.user.locale || "" },
+              language: { locale: booking.user.locale || "en" },
             }
           : { name: "", email: "", timeZone: "", language: { locale: "" } },
```

Comment:
@CarinaWolli Should this too be locale: ""?
```suggestion
          : { name: "", email: "", timeZone: "", language: { locale: "en" } },
```